<!DOCTYPE html>.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Cooking up a Storm</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+      <header class="header-background">
+        <h1>Cooking up a Storm</h1>
+        <nav class = "navibar">
+            <a href= "<!DOCTYPE html>.html">HOME</a>
```

Comment:
This is a weird file name. It probably doesn't cause any issues, but naming it something like home.html or index.html would be better for readability.